src/sentry/feedback/usecases/create_feedback.py

```diff
@@ -142,8 +142,15 @@ def fix_for_issue_platform(event_data):
     # If no user email was provided specify the contact-email as the user-email.
     feedback_obj = event_data.get("contexts", {}).get("feedback", {})
     contact_email = feedback_obj.get("contact_email")
-    if not ret_event["user"].get("email", ""):
-        ret_event["user"]["email"] = contact_email
+
+    if contact_email:
```

Comment:
```suggestion
```
Sorry I know I suggested this condition but I think we should remove it, None is allowed here and is potentially useful for the frontend. For ex if we wanted to filter on/search for feedbacks w/no email.

Reply:
should we still keep the condition for the tag though? doesn't make sense to have `None` as a tag

Reply:
event.schema.json says null values are allowed

Reply:
okay removed the check completely